docs/shadow_config_spec.md

```diff
@@ -675,30 +676,30 @@ environment: { ENV_A: "1", ENV_B: foo }
 
 #### `hosts.<hostname>.processes[*].expected_final_state`
 
-Default: !exited 0
-Type: !exited \<Integer\> OR !signaled \<Signal String or Integer\> OR !running
+Default: \{exited: 0\}  
+Type: \{exited: \<Integer\>\} OR \{signaled: [Unix Signal](./shadow_config_overview.md#unix-signals)\} OR "running"
```

Comment:
nit: I think if we're using quotes around "running", we should also use quotes around "exited" and "signaled" which are also strings.

Reply:
done